src/game/WorldHandlers/SpellEffects.cpp

```diff
@@ -2507,7 +2507,7 @@ void Spell::EffectPickPocket(SpellEffectIndex /*eff_idx*/)
     {
         int32 chance = 10 + int32(m_caster->getLevel()) - int32(unitTarget->getLevel());
 
-        if (chance > irand(0, 19))
```

Comment:
Hum, why was it 19? and where did you find the value of 5 ? a part from that everythign seems ok to me